src/drivers/distance_sensor/tfmini/TFMINI.cpp

```diff
@@ -212,8 +407,34 @@ TFMINI::collect()
 		return -EAGAIN;
 	}
 
+	int8_t signal_quality;
```

Comment:
Perhaps you could initialize signal quality here and omit it from the default case.